### DISCUSSION TITLE
docs/web/request-context

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#86] [General]
# docs/web/request-context

在GoFrame框架中使用Context对象处理请求流程中的上下文变量共享。通过提供必要的方法，开发者可以在请求开始时设置自定义变量，并在后续的处理中访问它们。此外，本文包含了如何集成第三方组件以增强功能的示例代码和详细步骤。

https://goframe.org/docs/web/request-context

**Comment by Lwxjjr** [comment]
这里的输出没有写，编写者可能漏了吧

**Reply by gqcn** [reply]
Thanks for feedback, fixed.
